.github/workflows/visual-regression-test.yml

```diff
@@ -77,7 +77,6 @@ jobs:
           --outDir packages/spindle-ui/.reg-suit/actual_images/
           --serverCmd 'npx serve -p 5000 packages/spindle-ui/public'
           http://localhost:5000
-      - run: git checkout ${{ github.head_ref || github.ref_name }}
```

Comment:
This line is probably necessary for reg-suit:
https://github.com/reg-viz/reg-suit#workaround-for-detached-head

Reply:
Ah. Let's give the step a name so people understand its importance.